mypy/indirection.py

```diff
@@ -22,9 +22,10 @@ def __init__(self) -> None:
         self.cache = {}  # type: Dict[types.Type, Set[str]]
 
     def find_modules(self, typs: Iterable[types.Type]) -> Set[str]:
-        return self._visit(*typs)
+        return self._visit(typs)
 
-    def _visit(self, *typs: types.Type) -> Set[str]:
+    def _visit(self, typ_or_typs: Union[types.Type, Iterable[types.Type]]) -> Set[str]:
```

Comment:
Why not just change callers passing a single type to do `self._visit([typ])`, so we don't need the union type?

Reply:
That's what I did at first but I felt that it uglified the code more than I liked. I'd be happy to go back to that though if people think it is nicer.

Reply:
I am fine with keeping this a union.